hardware/doc/Bitstream_flashing.md

```diff
@@ -49,7 +56,10 @@ The output bitstream file names will have `_FACTORY` appended.
 # Initial programming of a blank or bricked card
 
 There are two ways to program a card from scratch. Both ways require a Xilinx Platform Cable USB II.  
-Connect the Platform Cable with the card as described in the card's reference manual. The Alpha-Data KU3 has an on-board connector for the Platform Cable, the Nallatech 250S requires a Development & Debug Breakout Board.
+Connect the Platform Cable with the card as described in the card's reference manual. 
```

Comment:
>> Both ways require a Xilinx Platform Cable USB II  

is no longer true for the 8K5 which has its built-in programmer logic.
=> Update of the whole paragraph needed
Typo: addional=>additional